### ISSUE TITLE
Wrong expectation in co19/Language/Expressions/Shift/integer_t04/*

### ISSUE THREAD

**Issue by johnniwinther** [dart-lang/co19#1015]
The tests
```
co19/Language/Expressions/Shift/integer_t04/01
co19/Language/Expressions/Shift/integer_t04/02
co19/Language/Expressions/Shift/integer_t04/03
co19/Language/Expressions/Shift/integer_t04/04
co19/Language/Expressions/Shift/integer_t04/05
co19/Language/Expressions/Shift/integer_t04/06
co19/Language/Expressions/Shift/integer_t04/07
```
wrongfully expect compile time errors for negative triple-shift argument. This should assume runtime errors instead.

**Comment by iarkh** [comment]
Fixed with 5c5c516213eafa519021705bd5057afcb40df18f

**Comment by alexmarkov** [comment]
@iarkh Do we need a separate fix for `co19_2`?
